http-gateway/web/src/containers/pending-commands/_pending-commands-list.js

```diff
@@ -0,0 +1,333 @@
+import { useEffect, useMemo, useState } from 'react'

```

Comment:
why are you naming the files with _ at the beginning, a new convention?

Reply:
Yes, kinda. Smaller parts are with _ and the containers/pages are without it.